include/rapidcheck/state/Command.h

```diff
@@ -27,6 +27,8 @@ class Command {
   /// properly.
   virtual void run(const Model &s0, Sut &sut) const;
 
+  virtual std::function<void(const ModelT&)> run(Sut &sut) const;
```

Comment:
Took a stab at this in 8e3726bd3abbdbfdd76370167aae2aa034857dcd. But I'm too pleased with the writing. Do you have any ideas?